src/events/pubsub-http.ts

```diff
@@ -21,7 +21,7 @@ const IncomingMessagePayloadSchema = z.object({
 });
 
 export function createIncomingPubSubMessageHandler(
-    events: PubSubEvents,
+    pubSubEvents: PubSubEvents,
```

Comment:
We could have just renamed this to `commandBus` - but that's not quite right. Because the type of `commandBus` is technically just `AsyncEvents` - which doesn't confirm to the `PubSubEvents` needed here.

When the wrong type is injected, we get a runtime error which is tricky to track down. But here `pubSubEvents` is registered conditionally so we will get an error on boot if things are misconfigured.

Reply:
I don't want to block this PR - just a thought! Think we should get a fix in and your solution works 👍